### DISCUSSION TITLE
Renault Kadjar RLink2 - DDT4ALL black screen recover

### DISCUSSION THREAD

**Discussion by BigAndy2201** [cedricp/ddt4all#1194] [Q&A]
Hi

With reference to the following page here https://github.com/cedricp/ddt4all/issues/567

I also had the same issue as testuer in the thread.  I accidentally clicked on the display setting in dev mode, and the screen went black.
Using DDT4ALL, I managed to recover the screen to have info on it again by following the same path as testeur did.  The display details were already there just needed to hit the write button.
When I reset, the screen was back working again, but unfortunately it is inverted!  See screenshot.

![inverted](https://github.com/user-attachments/assets/bb7d2bd8-0f31-4307-ad65-ccd0227fa285)

The settings testeur added:
![ddt4alldispl](https://github.com/user-attachments/assets/b754e112-415c-4d55-ac79-dfde8e314702)

What do I need to change in DDT4All to invert the screen back to the correct direction please?  :)

**Comment by shrlnm** [comment]
@BigAndy2201 
https://gitlab.com/py_ren/pyren/-/blob/pyren3/pyren3/macro/rlink2/rlink2-rotate-screen.cmd?ref_type=heads
